pallets/teebag/src/benchmarking.rs

```diff
@@ -0,0 +1,444 @@
+use super::{Pallet as Teebag, *};
+use frame_benchmarking::v2::*;
+use frame_support::assert_ok;
+use frame_system::RawOrigin;
+use hex_literal::hex;
+
+fn assert_last_event<T: Config>(generic_event: <T as Config>::RuntimeEvent) {
+	frame_system::Pallet::<T>::assert_last_event(generic_event.into());
+}
+
+fn create_test_enclaves<T: Config>(n: u32, mrenclave: MrEnclave) {
+	for i in 0..n {
+		let who: T::AccountId = account("who", i, 1);
+		let test_enclave = Enclave::new(WorkerType::Identity).with_mrenclave(mrenclave);
+		assert_ok!(Teebag::<T>::add_enclave(&who, &test_enclave));
+	}
+}
+
+fn generate_test_mrenclave(index: u64) -> MrEnclave {
+	let seed: u64 = 1671606747000 + index;
+	let mut mrenclave = [0u8; 32];
+	for (i, byte) in seed.to_ne_bytes().iter().cycle().take(32).enumerate() {
+		mrenclave[i] = *byte;
+	}
+
+	MrEnclave::from(mrenclave)
+}
+
+fn create_test_authorized_enclaves<T: Config>(n: u32, worker_type: WorkerType) {
+	for i in 0..n {
+		let mrenclave = generate_test_mrenclave(i as u64);
+		AuthorizedEnclave::<T>::try_mutate(worker_type, |v| v.try_push(mrenclave))
+			.expect("Failed to add authorized enclave");
+	}
+}
+
+const PUBKEY: [u8; 32] = [
```

Comment:
Could you leave some comments how are these `const`s generated? From some test-data, or copied from somewhere?